app/src/main/java/com/bogdan/codeforceswatcher/features/users/UserActivity.kt

```diff
@@ -105,11 +106,16 @@ class UserActivity : AppCompatActivity() {
         }
 
         for (ratingChange in user.ratingChanges) {
-            val ratingUpdateTime = ratingChange.ratingUpdateTimeSeconds.toFloat()
-            val newRating = ratingChange.newRating.toFloat()
-            val data = ratingChange.contestName
-
-            entries.add(Entry(ratingUpdateTime, newRating, data))
+            ratingChange.run {
```

Comment:
It should be rather `with(ratingChange)`. The same functionality, but another syntax meaning.

Reply:
Also it can be a good exercise for you: try to extract this as an extension method for ratingChange: `func RatingChange.toChartItem() { ChartItem() }`.